sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -8,6 +8,7 @@ import isBuffer from "is-buffer";
 import { Buffer } from "buffer";
 import * as Constants from "../util/constants";
 import { AbortError, AbortSignalLike } from "@azure/abort-controller";
+import { getPlatformSpecificData } from "@azure/core-http/types/latest/src/policies/msRestUserAgentPolicy";
```

Comment:
`getPlatformSpecificData()` uses `os`. Doesnt that conflict with our need to work in the browser?

Reply:
Removed, as I said mentioned here https://github.com/Azure/azure-sdk-for-js/pull/10092#discussion_r456626013